Cargo.toml

```diff
@@ -25,7 +25,7 @@ members = [
   "partiql-rewriter",
   "partiql-types",
   "partiql-value",
-
+  "partiql-visualize-dot",
```

Comment:
Could you put this new crate under extensions? Similar to what was done for `partiql-extension-ion` and `partiql-extension-ion-functions`?

Reply:
I've moved the crate to `extension` and renamed it to `partiql-extension-visualize`